main.py

```diff
@@ -37,7 +39,7 @@ def reEmail(fname):
 def reAward(fname):
   fhand = open(fname, 'r')
   for line in fhand:
-    extr = re.findall('.*', line)
+    extr = re.findall('-(.*)', line)
```

Comment:
This will extract extra space in front of the award winner.  "  Usain Bolt" 

`extr = re.findall('- (.*)', line)`